README.md

```diff
@@ -1,2 +1,75 @@
-# vision-mission-structure
-Develop and communicate the academy's mission, vision, and structure
+# Mission / Vision Proposal
+
+Here I (Sam) will propose a new version of the mission / vision for the academy. I aim for the mission to carry us and 
+stay relevant for about 5 years and the vision to stay relevant for about 10 years.
+
+## Strategy 
+
+The strategy should focus on knowledge transfer. Producing world-class material that helps people improve 
+their lives is a good way to build an effective community.
+
+In terms of implementation, it is proposed to:
+
+Create a new organization called the Portugal Data Science Academy (PDSA) that adopts the Vision laid out in this document.
```

Comment:
Is Portugal Data Science Academy too restrictive? Should we try a more generic name to allow for international expansion? Or should we really zero in on our current market? 

If there is the remote possibility, in the foreseeable future, to open a chapter in another country we should carefully consider this